client/src/components/Connections/InputConnection.tsx

```diff
@@ -4,21 +4,16 @@ const InputConnection = () => {
   const [user1, setUser1] = useState("");
   const [user2, setUser2] = useState("");
 
-  const onSubmitForm = async (e:any) => {
+  const onSubmitForm = (e:any) => {
     e.preventDefault();
-    try {
-      const response = await fetch("http://localhost:3000/connections" + "/" + user1 + "/" + user2, {
+      const response = fetch("http://localhost:3000/connections" + "/" + user1 + "/" + user2, {
         method: "POST",
         headers: { "Content-Type": "application/json" },
       });
+
+      response.then((res) => {
```

Comment:
What will happen if the promise fails now that we have removed the `try/catch`?  `Promise.prototype.then()` only works for resolved promises and we still want to know that it has failed (log the error)

Also as a note, if you are just going to use the promise const to call it's methods you can do it directly:

```
fetch(...).then(...);
```